Although a true xterm has supported the ECMA-48 REP (repeat character) control
since January 1997, many other terminal emulators that claim to be xterm do
not support this. (For example, VTE, konsole, PuTTY, iTerm2).

For least surprise, remove this capability from
the xterm terminal definition in OmniOS.

See also http://invisible-island.net/ncurses/ncurses.faq.html#xterm_generic

diff -wpruN '--exclude=*.orig' a~/misc/terminfo.src a/misc/terminfo.src
--- a~/misc/terminfo.src	1970-01-01 00:00:00
+++ a/misc/terminfo.src	1970-01-01 00:00:00
@@ -4985,14 +4985,14 @@ xterm-xfree86|xterm terminal emulator (X
 xterm+nofkeys|building block for xterm fkey-variants,
 	npc,
 	kcbt=\E[Z, kent=\EOM, nel=\EE, use=ecma+index,
-	use=ansi+rep, use=ecma+strikeout, use=vt420+lrmm,
+	use=ecma+strikeout, use=vt420+lrmm,
 	use=xterm+sm+1006, use=xterm+tmux, use=ecma+italics,
 	use=xterm+keypad, use=xterm-basic,
 
 xterm-p370|xterm patch #370,
 	npc,
 	kcbt=\E[Z, kent=\EOM, nel=\EE, use=ecma+index,
-	use=ansi+rep, use=ecma+strikeout, use=xterm+pcfkeys,
+	use=ecma+strikeout, use=xterm+pcfkeys,
 	use=xterm+nofkeys, use=bracketed+paste,
 
 xterm-p371|xterm patch #371,
